.github/workflows/static.yml

```
@@ -2,27 +2,32 @@ name: Static Analysis
-        uses: shivammathur/setup-php@v2
+        uses: shivammathur/setup-php@f3e473d116dcccaddc5834248c87452386958240 # 2.37.2
         with:
-          php-version: 8.4
+          php-version: '8.4'
           coverage: none
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Install Composer dependencies
-        uses: ramsey/composer-install@v3
+      - name: Install composer dependencies
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)